Dockerfile.2.K8S

```diff
@@ -10,6 +10,9 @@ ARG KUBE_VERSION=1.7.8
 ARG KOPS_VERSION=1.7.0
 ARG HELM_VERSION=2.6.2
 
+# Install ssh (Required to perform port forwarding through bastion)
+RUN apk add openssh
```

Comment:
add --no-cache to not keep cache on the image

Reply:
Ça change quoi ? Je croyais que c’était simplement pour dire de ne pas utiliser la cache.

Reply:
ça change qu'il utilise pas de cache, donc il le telecharge pas. Dans son cas je suppose que ça fonctionne car il hérite du cache que tu crée dans la base

Reply:
Il n'est pas dans la cache, mais l'option --no-cache sert normalement à bypasser la cache et forcer à aller sur le network.

Reply:
Oui bypasser le cache local,  pour ne pas avoir a faire rm -rf /var/cache/apk/* à la fin pour alleger l'image. 
Dans ton docker de base tu fais RUN apk update. Ce qui veut dire que tu crée un cache local. Toute les autres commande apk vont chercher dans ce cache car ils partent de cette image.

Reply:
Donc, le problème n'est pas juste de mettre --no-cache, il faut aussi cleaner la cache dans le docker de base si je comprends bien.

Reply:
La bonne pratique est d'utiliser le --no-cache partout